cmd/kube-scheduler/app/server.go

```diff
@@ -167,10 +167,12 @@ func Run(ctx context.Context, cc *schedulerserverconfig.CompletedConfig, sched *
 	defer cc.EventBroadcaster.Shutdown()
 
 	// Setup healthz checks.
-	var checks []healthz.HealthChecker
+	var checks, readyzChecks []healthz.HealthChecker
```

Comment:
How about livezChecks? Just corresponding to readyzChecks.

Reply:
There are no checks for `/livez` (pingCheck is included by default) so it's unnecessary to define a variable for it here. Also see `healthz.InstallLivezHandler(pathRecorderMux)` in the line 291.